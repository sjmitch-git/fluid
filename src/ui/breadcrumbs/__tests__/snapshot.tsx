import { render } from '@testing-library/react'
import { Breadcrumbs } from '..'

it('renders Breadcrumbs unchanged', () => {
	const { container } = render(<Breadcrumbs></Breadcrumbs>)

	expect(container).toMatchSnapshot()
})
