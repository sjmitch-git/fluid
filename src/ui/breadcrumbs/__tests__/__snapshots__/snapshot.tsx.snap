// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Breadcrumbs unchanged 1`] = `
<div>
  <nav
    aria-label="breadcrumb"
    class="breadcrumbs group mx-auto w-full text-dark dark:text-light text-base "
    data-testid="breadcrumbs"
  >
    <ol
      class="flex flex-col md:flex-row"
    />
  </nav>
</div>
`;
